README.md

```diff
@@ -21,15 +21,15 @@ Es folgen zwei Tabellen mit Vorschlägen für den täglichen Gebrauch.
 | clone       | clonen             | nachmachen            |
 | fetch       | fetchen            | holen                 |
 | branch      | branchen           | abzweigen             |
-| commit      | commiten           | festlegen             |
+| commit      | commiten           | einbuchen             |
 | rebase      | rebasen            | (neu) erden           |
 | diff        | diffen             | unterscheiden         |
 | merge       | mergen             | zusammenführen        |
 | fork        | forken             | abspalten             |
 | stash       | stashen            | verstecken            |
 | tag         | taggen             | markieren             |
-| cherry-pick | cherry-picken      | Rosinen herauspicken  |
-| checkout    | checkouten         | nehmen                |
+| cherry-pick | cherry-picken      | Kirschen herauspicken |
```

Comment:
Sollte es nicht besser `Kirschen pflücken` in der Übersetzung heißen?

Reply:
Wenn ich mich richtig erinnere wurde Rosinen herauspicken gewählt, weil es eine Redensart ist (https://de.wiktionary.org/wiki/sich_die_Rosinen_herauspicken), ähnlich wie das "cherry-picking", daher würde ich da die direkte Übersetzung eher der Bedeutung unterordnen...